spec/factories/unavailabilities.rb

```diff
@@ -0,0 +1,12 @@
+FactoryGirl.define do
+  factory :unavailability do
+    sunday 'MyString'
```

Comment:
This factory should only contain the attributes necessary to create a valid `Unavailability`